### PR TITLE
scalasca/231

### PR DESCRIPTION
**Pull request by jgphpc**
review please.

### CODE REVIEW

**Comment by jgphpc** [comment]
need to fix Cube's plugin conflict with Qt...

**Comment by jgphpc** [comment]
ready to be merged

**Comment by jgphpc** [comment]
see also https://github.com/eth-cscs/production/pull/8

File: jenkins-builds/6.0.UP02-2016.11-gpu [hunk 1]
```diff
@@ -29,3 +29,6 @@
  Score-P-3.0-CrayIntel-2016.11.eb
  Score-P-3.0-CrayPGI-2016.11.eb
  VASP-5.4.1-CrayIntel-2016.11-cuda-8.0.eb
+ Scalasca-2.3.1-CrayGNU-2016.11.eb
```
Comment (gppezzi) [inline jenkins-builds/6.0.UP02-2016.11-gpu]: please respect the alphanumerical order

**Pull request merged**